### ISSUE TITLE
[App]: Tensorboard histograms don't get into wandb properly

### ISSUE THREAD

**Issue by zplizzi** [wandb/client#3436]
### Current Behavior

I am using the `sync_tensorboard=True` feature in `wandb.init`, with a script that logs data using the tensorboard. When I log a histogram into the tensorboard, it shows up in the tensorboard tab (where an actual tensorboard instance is running) of wandb, but not in the main wandb run metrics view ("charts" view) as it should.

Here's a run you can look at to investigate this behavior: https://wandb.ai/sourceress/zack_sb3/runs/14paycf8. Note the key `train/advantages_hist` is in the tensorboard view but not in the charts view.

### Environment

OS: Mac

Browsers: Chrome

Version:

**Comment by zplizzi** [comment]
Hmm, in another run, some of them worked - not sure what happened in the first run I linked. Here's the one where some hists came thru: https://wandb.ai/sourceress/zack_sb3/runs/1yzyb4j0

**Comment by zplizzi** [comment]
Oh, I think I might have figured out the issue - found this warning buried under a lot of unrelated logging: 

`wandb: WARNING Not logging key "train/value_target_hist". Histograms must have fewer than 512 bins`

I guess I could modify the tensorboard logging code to ensure it uses less than 512 bins, although tbh the whole reason this feature (`sync_tensorboard=True`) is useful is because I can get wandb logging out of someone else's code without digging into + modifying their code. So it would be sweet if there was a graceful automatic fallback here. In theory a histogram could be automatically resampled down to a lower number of bins?

**Comment by vanpelt** [comment]
Yep, totally true in theory but it's non-trivial which is why we punted on it originally.  We'll make sure this is a big +1 for actually implementing the resampling.

**Comment by younik** [comment]
Trying to avoid this issue, I used `np.digitize`, in this way:
```python
bins = np.linspace(np.min(values), np.max(values), num=512)
digitized_values = np.digitize(values, bins=bins)
writer.add_histogram(tag, bins[digitized_values - 1])
```

but I still get `wandb: WARNING Not logging key "train/value_target_hist". Histograms must have fewer than 512 bins`
